janus/utils/enums.py

```diff
@@ -474,6 +474,12 @@ class EmbeddingType(Enum):
         "url": "https://github.com/tree-sitter/tree-sitter-php",
         "example": "<?php\n// Hello, World!\necho 'Hello, World!';\n",
     },
+    "plantuml": {
+        "comment": "'",
+        "suffix": "puml",
+        "url": "",
```

Comment:
Do we need to point to a Git repo with a tree-sitter PlantUML grammar?

Reply:
since plantuml is only used as an output language, I specifically left this blank to indicate any scenario where it is being used as an input language is out of scope. 
In addition, there doesn't seem to be a native tree-sitter parser for plantuml (or any uml language for that matter) so if for some reason we wanted to do this it would require a hacky solution like our C++ fix.

Reply:
Got it!